### ISSUE TITLE
Course-Add step-Add Test

### ISSUE THREAD

**Issue by sjkadali** [open-learning-exchange/planet#6430]
In Course, click Add step and then Add Test. If nothing is entered and Return button clicked, message saying "New Test added" is shown. 
![Screenshot from 2020-05-28 10-27-07](https://user-images.githubusercontent.com/39105848/83155164-f7d72280-a0ce-11ea-8fe0-34207469fbd7.png)

**Comment by lmmrssa** [comment]
Yes because exam is already created but empty one. We just don't show empty exam anywhere so you don't see any icon
